cdap-ui/app/cdap/services/WizardStores/AddNamespace/ActionCreator.js

```diff
@@ -17,6 +17,14 @@ import AddNamespaceStore from 'services/WizardStores/AddNamespace/AddNamespaceSt
 import {MyNamespaceApi} from 'api/namespace';
 
 const PublishNamespace = () => {
+  return publishOrEditNamespace(MyNamespaceApi.create);
+};
+
+const editNamespaceProperties = () => {
```

Comment:
nit: Should this be `EditNamespaceProperties`? to be consistent with `PublishNamespace`? Or change `publishNamespace` or `createNamespace`